app/src/main/java/com/orgzly/android/sync/SyncWorker.kt

```diff
@@ -84,15 +89,28 @@ class SyncWorker(val context: Context, val params: WorkerParameters) :
 
         checkConditions()?.let { return it }
 
+        val syncStartTime = System.currentTimeMillis()
+
         syncRepos()?.let { return it }
 
         RemindersScheduler.notifyDataSetChanged(App.getAppContext())
         ListWidgetProvider.notifyDataSetChanged(App.getAppContext())
         SharingShortcutsManager().replaceDynamicShortcuts(App.getAppContext())
 
+        val syncEndTime = System.currentTimeMillis()
+
         // Save last successful sync time to preferences
-        val time = System.currentTimeMillis()
-        AppPreferences.lastSuccessfulSyncTime(context, time)
+        AppPreferences.lastSuccessfulSyncTime(context, syncEndTime)
+
+        if (LogMajorEvents.isEnabled()) {
```

Comment:
Are there also minor events? 
Sync does not sound like a major event but without reviewing the code in detail it is hard to say.

Reply:
Yeah, good question. I believe the "log major events" feature is a misnomer, and that it was added purely out of a need to debug reminders. A lot of details about reminders is logged during every sync, which is hardly a "major event", as you point out.